jest.config.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
   transformIgnorePatterns: ['<rootDir>/node_modules/'],
   verbose: false,
   moduleNameMapper: {
-    '^.+\\.(css|less|scss)$': 'babel-jest',
-    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/__mocks__/fileMock.js',
```

Comment:
Hello @Amoswachira and @Bakley ,
When the regex is removed, static files like images would throw errors during testing. 
For it to work, you should restore the `__mocks__` directory and the `__mocks__/fileMock.js` file that you have deleted